stage1/init/init.go

```diff
@@ -122,8 +122,18 @@ func getArgsEnv(p *Pod, debug bool) ([]string, []string, error) {
 
 	case "src":
 		args = append(args, filepath.Join(common.Stage1RootfsPath(p.Root), nspawnBin))
-		args = append(args, "--boot") // Launch systemd in the pod
-
+		args = append(args, "--boot")              // Launch systemd in the pod
+		args = append(args, "--register", "false") // We cannot assume the host system is running systemd
```

Comment:
:+1: 

When adding `--register`, `stage1/rootfs/usr_from_src/patches/0001-nspawn-just-disable-registration-on-non-sysd-hosts.patch` would not need to hardcode `arg_register = false;` anymore so this can be replaced by the [upstream patch](http://cgit.freedesktop.org/systemd/systemd/commit/?id=4f923a1984476de3441922ee5bf7102ebdd250ef). Can you check if replacing our systemd patch by the upstream patch work when ``--register`` is added?

This will need to be revisited when https://github.com/coreos/rkt/issues/438 gets fixed. Can you add a comment such as:
```
// TODO: integrate rkt with the host systemd when available
// See https://github.com/coreos/rkt/issues/438
```